include/RAJA/util/PluginInit.hpp

```diff
@@ -0,0 +1,20 @@
+#ifndef RAJA_Plugin_Init_HPP
+#define RAJA_Plugin_Init_HPP
+
+#include <string>
+#include <vector>
+
+namespace RAJA {
+namespace plugin {
+
+std::vector<std::string> paths;
+
+void init(const std::string& path)
+{   
+    paths.push_back(path);
```

Comment:
I think it would be better if this looped over the plugins and called an "init" on each one. (avoiding the global "paths" variable).